spec/rubocop/cop/style/sole_nested_conditional_spec.rb

```diff
@@ -663,6 +663,46 @@ def foo
             end
         RUBY
       end
+
+      context 'with a `csend` node' do
+        it 'registers an offense and corrects' do
+          expect_offense(<<~RUBY)
+            if foo
+              if bar&.baz quux
+              ^^ Consider merging nested conditions into outer `if` conditions.
+                do_something
+              end
+            end
+          RUBY
+
+          expect_correction(<<~RUBY)
+            if foo && (bar&.baz quux)
```

Comment:
It would be more better to correct as follows.

```suggestion
            if foo && bar&.baz(quux)
```

Reply:
I agree, but this is the existing autocorrection for `send` nodes I'm just extending it to `csend` and `block` nodes as well to prevent syntax errors from autocorrection.

Would you be good with accepting this as-is and improving autocorrection in a future PR?

Reply:
OK, I think we can do it separately. It would be useful to leave a `FIXME` comment.

Reply:
Added!